webpack.functions.js

```diff
@@ -0,0 +1,8 @@
+const webpack = require('webpack')
```

Comment:
What is this global `GENTLY` for? Why is it only needed for the `serve` npm script?

Reply:
I should put a comment here.

This is needed to import crypto in `auth.js`

Found the solution here:
https://github.com/netlify/netlify-lambda/issues/64

Reply:
I just read the thread but don't understand what the issue is all about. lol
One thing I noticed is that `netlify-lambda serve` is deprecated and they provide something else.
If it is a working solution, I am fine with this for now but can you move this file into the `fucnitons` directory because this Webpack config is only specific to Netlify Lambda.

In addition, I would double check if it is okay to put this config option to `netlify-lambda serve` when `netlify-lambda build` is the actual command that Netlify runs during deployment. `netlify-lambda serve` is only for local development.

Reply:
I _believe_ the fix is only needed for local development. I will try to confirm.